pkg/3scale/amp/component/s3.go

```diff
@@ -14,6 +14,11 @@ const (
 	S3SecretAWSSecretAccessKeyFieldName = "AWS_SECRET_ACCESS_KEY"
 )
 
+const (
+	StorageServiceEndpointScheme = "STORAGE_SERVICE_ENDPOINT_SCHEME"
+	StorageServiceEndpointHost   = "STORAGE_SERVICE_ENDPOINT_HOST" // TODO Includes Host and Port. Maybe choose a better name?
```

Comment:
```suggestion
	StorageServiceEndpointHost   = "AWS_HOSTNAME" // TODO Includes Host and Port. Maybe choose a better name?
```

See https://github.com/3scale/3scale-operator/pull/302/files?file-filters%5B%5D=.go#r365307907

Reply:
See my comment above